src/test/java/be/seeseemelk/mockbukkit/entity/EntityMockTest.java

```diff
@@ -1051,4 +1051,10 @@ void testSetGravity()
 		assertFalse(entity.hasGravity());
 	}
 
+	@Test
+	void getEntityId()
+	{
+		assertTrue(entity.getEntityId() > 0);
+	}
```

Comment:
This should assert a specific value, not just greater than one.

Please also add tests to ensure the id gets incremented correctly when new entities are added.

Reply:
I think you misunderstood my request. Something like the following would do

```java
@Test
void getEntityId_ProperlyIncrements()
{
    assertTrue(entity.getEntityId() == 1);
    new SimpleEntityMock(server);
    assertTrue(entity.getEntityId() == 2);
}
```

Reply:
In that case, I've added [a test for that](https://github.com/MockBukkit/MockBukkit/pull/692/files#diff-72a20149babc2d066c370bb1f0f370d4d45906a999e81cbe2f3cc4cc66948638R1062).